README.rst

```diff
@@ -1,17 +1,18 @@
 ==================================
-XBlock flow-control |build-status|
+XBlock Flow Control |build-status|
 ==================================
 
-The Flow Control Xblock provides a way to display the content of a unit or to redirect the user elsewhere based on compliance with a condition that evaluates the submission or the score of a problem or a set of problems.
+The Flow Control XBlock provides a way to display the content of a unit or to redirect the user elsewhere based on compliance with a condition that evaluates the submission or the score of a problem or a set of problems.
 
 Installation
 ------------
 
-Install the requirements into the Python virtual environment of your `edx-platform` installation by running the following command from the root folder:
+Inside both LMS and Studio shells, using `make lms-shell` and `make studio-shell` in your devstack directory:
```

Comment:
I think that these changes should be in another PR, if we are going to add this, then the title should have to change to something more related to installation on the devstack.

Reply:
Changed to:

Installing on Open edX Devstack
------------